build.cmd

```diff
@@ -0,0 +1,62 @@
+@echo off
+setlocal
+
+rem 1. Set the following for the options you want to build.
+set CUDNN=true
+set DX12=false
+set OPENCL=false
+set MKL=false
+set DNNL=false
+set OPENBLAS=false
+set EIGEN=false
+set TEST=false
+
+rem 2. Edit the paths for the build dependencies.
+set CUDA_PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0
```

Comment:
in my experience VS2019 + cuda v10.0 didn't work so well - I switched to 10.1

Reply:
Is this the usual cuda header version check? I can add it to the readme files.

Reply:
Yeah I think it was that.

Reply:
Looks good.